pagopa/io-app
# feat(Sicilia Vola): [IASV-30] Additional information screen for disabled

## Short description
This PR add the additional information screen in the voucher generation flow when the selected category is disabled

## List of changes proposed in this pull request
- Added `DisabledAdditionalInfoScreen`

https://user-images.githubusercontent.com/11773070/128034032-6ba289ad-cee1-454a-b033-4cf336e173ca.mp4


## How to test
Modify in the `CheckStatusRouterScreen` the following lines:
https://github.com/pagopa/io-app/blob/0150ad3c78a7b842f02d929ab88ca5b43ce0bf23/ts/features/bonus/siciliaVola/screens/voucherGeneration/CheckStatusRouterScreen.tsx#L99-L100
in:
```
isServiceAlive: remoteReady(true),
tosAccepted: remoteReady(true)
```
Starts the flow from the SiciliaVola service detail and select in:
- `CheckStatusRouterScreen` -> `Resident in Sicily`
- `SelectBeneficiaryCategoryScreen` -> `Disabled`

Make sure you have the updated version of the master branch on dev-server in order to see the service details.
